site/pages/react-native/signer/expo-installation.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: React Native Signer Expo Installation
+description: Setup the React Native Signer Package in your Expo Project
+---
+
+:::warning
+To use the signer, your project has to have the new architecture enabled. Refer to the [Expo Documentation](https://docs.expo.dev/guides/new-architecture/) for more information.
+It is recommended to be on the latest Expo SDK version (version 52 or newer) to ensure compatibility.
+:::
+
+:::info
+This package is not supported on Expo Go. If your app is currently running on Expo Go, you would have to create a development build to use the signer. Check out [Expo Development Builds](https://docs.expo.dev/develop/development-builds/create-a-build/) for more information.
+:::
+
+# Setting up the React Native Signer in an Expo Project
+
+This guide assumes you have an existing Expo project, running on the [new architecure](https://reactnative.dev/blog/2024/10/23/the-new-architecture-is-here) and have already followed the steps in the [configuration guide](/react-native/signer/configuration-guide) to properly configure your Account Kit project.
+
+To create a new Expo project, please refer to the [Expo Installation Guide](https://docs.expo.dev/get-started/create-a-project/).
+
+## Install the React Native Signer Package
+
+To install the React Native Signer package, run the following command:
+
+:::code-group
+
+```bash [npm]
+npm install @account-kit/react-native-signer
+```
+
+```bash [yarn]
+yarn add @account-kit/react-native-signer
+```
+
+:::
+
+## Add Supporting Dependencies
+
+To ensure the signer works properly, you'll need to add the following dependencies to your project:
+
+:::code-group
+
+```bash [npm]
+npm install @account-kit/signer react-native-mmkv zustand abitype
```

Comment:
woah... I wonder why RN doesn't pull in these deps (specifically the account-kit/signer and zustand and abitype ones)

Reply:
Yeah it's quite odd tbh, warrants a deeper look into how metro loads up dependencies of dependencies.